v4/export/sql.go

```diff
@@ -315,6 +319,26 @@ func GetColumnTypes(db *sql.Conn, fields, database, table string) ([]*sql.Column
 	return rows.ColumnTypes()
 }
 
+func GetPrimaryKeyColumns(db *sql.Conn, database, table string) ([]string, error) {
+	priColQuery := "SELECT COLUMN_NAME FROM information_schema.KEY_COLUMN_USAGE " +
+		"WHERE TABLE_SCHEMA = ? AND TABLE_NAME = ? AND CONSTRAINT_NAME = 'PRIMARY' order by ORDINAL_POSITION;"
+	var cols []string
+	rows, err := db.QueryContext(context.Background(), priColQuery, database, table)
+        if err != nil {
+                return cols, withStack(errors.WithMessage(err, priColQuery))
```

Comment:
We have already updated this part. You can use `errors.Annotatef(err, "sql: %s", priColQuery)` instead.

Reply:
Hi, it is all done. Please review it again.